x-pack/filebeat/input/cel/input.go

```diff
@@ -484,6 +514,12 @@ func (i input) run(env v2.Context, src *source, cursor map[string]interface{}, p
 	return err
 }
 
+func updateStatus(reporter status.StatusReporter, state status.Status, msg string) {
+	if reporter != nil {
```

Comment:
Every input is going to need a variant of this, can you make this reusable? 

Could this be a method on v2.Context that includes the nil check and just does nothing in that case? Someone will eventually have their time wasted because this is nil-able so let's hide that if we can.

Rather than `updateStatus(env.StatusReporter, status.Failed, "failed to run: "+err.Error())` could it be `env.UpdateStatus(status.Failed, "failed to run: "+err.Error())`?